packages/palette/src/elements/ModalDialog/ModalDialog.tsx

```diff
@@ -26,7 +26,9 @@ export const ModalDialog: React.FC<ModalDialogProps> = ({
 }) => {
   const isMounted = useDidMount({ clearCallStack: true })
 
-  const [{ width, ...boxProps }, modalProps] = splitBoxProps(rest)
+  const [{ width, ...boxProps }, modalProps] = splitModalDialogContentProps(
+    rest
+  )
```

Comment:
maybe we don't need `splitModalDialogContentProps` here and it will be enough to do something like that. what do you think about this?

```javascript
const [
  { width, ...boxProps },
  { titleVariant, ...modalProps },
] = splitModalDialogContentProps(rest)
```